cmd/duffle/relocate.go

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
nit: can you clean those copyright headers? I don't think it will fit with duffle licence. @radu-matei ?

Reply:
@radu-matei The contributed code is copyright Pivotal and licensed under the Apache License, v2. What else does the duffle license require?

Reply:
Correction. My mistake. The contribution is covered by the MIT license and we are happy to conform to the project standard of not including copyright headers in source files.